pkg/ide/jetbrains.go

```diff
@@ -86,3 +91,30 @@ func isAlreadyDownloaded(projectHostname, downloadPath string) bool {
 	err := statCmd.Run()
 	return err == nil
 }
+
+func getJetbrainsVersion(productCode string) string {
+	jetbrainsDataServicesUrl := fmt.Sprintf("https://data.services.jetbrains.com/products/releases?code=%s&type=release&latest=true&build=", productCode)
+	res, err := http.Get(jetbrainsDataServicesUrl)
+	if err != nil {
+		panic("failed to request version for product")
```

Comment:
This function should not panic but return an error instead stat the `OpenJetbrainsIDE` function would return back as well.